packages/loader/container-utils/src/test/error.spec.ts

```diff
@@ -101,20 +101,20 @@ describe("Errors", () => {
         it("Copy message and stack", () => {
             const innerError = new LoggingError("hello");
             innerError.stack = "extra special stack";
-            const newError = wrapError(innerError, (message) => new LoggingError(message));
+            const newError = wrapError(innerError, (message) => (new LoggingError(message)) as LoggingError & { fluidErrorCode: "fluidErrorCode", "errorType": ContainerErrorType.genericError });
```

Comment:
i don't think we should need this casting. should LogginerError implement IFluidErrorBase

Reply:
It needs casting I think because of & operator. Otherwise it gives error. 
Logging error as per name feels like have properties related to logging like add and set telemetry properties apis and not other specific properties like messages, errorCode etc which are implemented by specific error classes. Don't have anhy hard opinion on this. But we can add it in future if we see.